Source/Raymarcher/Shaders/Private/WindowedSampling.usf

```diff
@@ -50,3 +50,33 @@ float4 SampleWindowedVolumeOctreeStep(int3 CurPos, float StepSize, Texture3D Vol
 	const float DataValue = Volume.Load(MipLevelPos, 0).r;
 	return SampleWindowedTransferFunction(DataValue, StepSize, TF, TFSampler, WindowingParams);
 }
+
+// Returns a bit number where all the bits between the window center and window width are ones.
```

Comment:
`Returns a bit mask...`

Reply:
Also, where is the TF -> Bitmask functionality? This just eliminates the stuff outside of the window, not transparent stuff within the window.